eventhubs/spring-cloud-azure-starter/spring-cloud-azure-sample-eventhubs-kafka/README.md

```diff
@@ -145,6 +142,41 @@ To destroy the resources you created.
 terraform -chdir=./terraform destroy -auto-approve
 ```
 
+### Use Azure Resource Manager to retrieve connection string
```

Comment:
Change to 
## Use Azure Resource Manager to Retrieve Connection String

1. we should use the same title level
2. we should format the title